src/frequenz/sdk/timeseries/_moving_window.py

```diff
@@ -163,6 +166,16 @@ def __init__(  # pylint: disable=too-many-arguments
             self._run_impl()
         )
 
+    @property
+    def sampling_period(self) -> timedelta:
+        """
+        Return the sampling period of the MovingWindow.
+
+        Returns:
+            The sampling period of the MovingWindow.
```

Comment:
Nitpicking :grimacing: 

```suggestion
        Return the sampling period of the `MovingWindow`.

        Returns:
            The sampling period of the `MovingWindow`.
```